### PR TITLE
run initial gapic

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**

### CODE REVIEW

File: google-cloud-vision/lib/google/cloud/vision/v1.rb [hunk 1]
```diff
@@ -17,78 +17,78 @@
...
+      # 2. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
+      # 3. [Enable the Cloud Vision API.](https://console.cloud.google.com/apis/api/vision)
+      # 4. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+      #
+      # ### Preview
+      # #### ImageAnnotatorClient
+      # ```rb
+      # require "google/cloud/vision"
+      #
+      # image_annotator_client = Google::Cloud::Vision::V1.new(version: :v1)
```
Comment (dazuma) [inline google-cloud-vision/lib/google/cloud/vision/v1.rb]: Argh... Can you manually fix this, and open an issue on gapic-generator? (It's supposed to be `Google::Cloud::Vision.new(version: :v1)`.)

File: google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb [hunk 2]
```diff
@@ -136,7 +136,7 @@ def initialize \
               updater_proc = credentials.updater_proc
             end
 
-            package_version = Gem.loaded_specs['google-cloud-vision'].version.version
+            package_version = Gem.loaded_specs['google-cloud'].version.version
```
Comment (TheRoyalTnetennba) [inline google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb]: Seems to be an naming issue in generation. It also generates a google-cloud.gemspec.

File: google-cloud-vision/lib/google/cloud/vision/v1/doc/overview.rb [hunk 3]
```diff
@@ -36,15 +36,15 @@ module Cloud
     #
     # ### Installation
     # ```
-    # $ gem install google-cloud-vision
+    # $ gem install google-cloud
```
Comment (dazuma) [inline google-cloud-vision/lib/google/cloud/vision/v1/doc/overview.rb]: Argh... What's going on?? This needs to be reverted (and probably another bug filed against gapic-generator).

File: google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb (hunk 2 above)
Comment (dazuma) [inline google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb]: Revert

File: google-cloud-vision/lib/google/cloud/vision/v1.rb (hunk 1 above)
Comment (dazuma) [reply google-cloud-vision/lib/google/cloud/vision/v1.rb]: Ohh... you know what, we can't make this change yet because this isn't gapic-only....

**Comment by dazuma** [comment]
Hmm, It looks like there are some issues. Maybe let's skip the gapic regen for now and just add the delegator method.

File: google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb (hunk 2 above)
Comment (TheRoyalTnetennba) [reply google-cloud-vision/lib/google/cloud/vision/v1/image_annotator_client.rb]: @dazuma Yeah, basically just wanted an easy way to share what's going on.

**Comment by blowmage** [comment]
@dazuma Should this PR be closed and reopened when the code generation issues are resolved?

**Pull request closed without merging**